test/sequential/test-performance-eventloopdelay.js

```diff
@@ -50,12 +49,19 @@ const { sleep } = require('internal/util');
   });
 }
 
+// Can't use `sleep` from internal/util because it doesn't do a busy wait
+// which means we can get 0 values in the histogram sometimes.
+function busySleep(ms) {
+  const target = Date.now() + ms;
+  while (Date.now() < target) {}
```

Comment:
I can't think of a reason to use `process.hrtime.bigint()` instead of `Date.now()` here but mentioning it in case someone else might have a reason.